test/TensorFlowRuntime/dynamic_compilation.swift

```diff
@@ -14,13 +14,6 @@ import StdlibUnittest
 
 var DynamicCompilationTests = TestSuite("DynamicCompilation")
 
-DynamicCompilationTests.testCPUOrGPU("Const") {
-  _RuntimeConfig.printsDebugLog = true
-  let x: TensorHandle<Float> = #tfop("Const", dtype$dtype: Float.tensorFlowDataType, value$tensor: Float(1.0))
```

Comment:
in case people do this, will we crash, or give a good error message? ideally should we have a test for that? if so pls consider filing a bug or adding a TODO.

Reply:
good point. we just crash now. I will file a bug.